OracleFPP/README.md

```diff
@@ -1,30 +1,18 @@
-# Oracle Fleet Patching and Provisioning (FPP) Vagrant boxes on VirtualBox or KVM/libVirt provider
+# Oracle Fleet Patching and Provisioning (FPP) Vagrant project on VirtualBox or KVM/libVirt provider
 
 ###### Author: Ruggero Citton (<ruggero.citton@oracle.com>) - Orale RAC Pack, Cloud Innovation and Solution Engineering Team
 
 This directory contains Vagrant build files to provision automatically
 one Grid Infrastructure and FPP Server host + (optional) an Oracle FPP target, using Vagrant, Oracle Linux 7.4 and shell scripts.
 ![](images/OracleFPP.png)
 
-## Prerequisites for VirtualBox
-1. Install [Oracle VM VirtualBox](https://www.virtualbox.org/wiki/Downloads)
-2. Install [Vagrant](https://vagrantup.com/)
-3. You need to download Database binary separately
-
-## Prerequisites for KVM/libVirt provider
-1. Install [KVM]/[libVirt]
-2. Install [Vagrant](https://vagrantup.com/)
-3. Install extra packages such ruby-devel libvirt-devel
-  - `yum install -y ruby-devel libvirt-devel`
-4. Install vagrant-libvirt as user
-  - `vagrant plugin install vagrant-libvirt`
-5. You need to download Database binary separately
-
-#### Note: *Using KVM/libVirt provider you may need to disable or manage host firewall to permit NFS traffic with the guest VMs*
-#### Note: if you are going to use KVM on OL7/OL8 please read 'https://blogs.oracle.com/linux/getting-started-with-the-vagrant-libvirt-provider-for-oracle-linux'
+## Prerequisites
 
+1. Read the [prerequisites in the top level README](../README.md#prerequisites) to set up either Vagrant with either VirtualBox or KVM
```

Comment:
```suggestion
1. Read the [prerequisites in the top level README](../README.md#prerequisites) to set up Vagrant with either VirtualBox or KVM
```

Reply:
note as the prerequisites are different based on the provider